src/components/analytics/events/transactionEvents.ts

```diff
@@ -58,6 +58,16 @@ export function approvalAnalytics(action: ApprovalAction, label?: string, value?
   })
 }
 
+type BundleAction = 'Send' | 'Sign' | 'Reject' | 'Error'
+export function bundleAnalytics(action: BundleAction, orderClass: OrderClass, label?: string, value?: number) {
+  sendEvent({
+    category: Category.BUNDLE,
+    action,
+    label: getClassLabel(orderClass, label),
+    value,
+  })
+}
+
 export type SwapAction = 'Send' | 'Error' | 'Reject'
```

Comment:
Because there's a category for claim, one for wrapping, one for swap.

Assuming we'll have more steps that can be eventually bundled, have a category only for it made sense.

But not a strong opinion, I'm fine using your proposal instead

Reply:
Agree with Anxo.
From a user perspective, it's a swap. The fact that we do approve it's a "side-effect".

Reply:
Fair enough, 2 vs 1, will update :)